add problem and solution documentation for pyenv not found

edithturn opened the pull request:
While I was following the instruction for the installation of pyenv, I found that adding 'eval "$(pyenv virtualenv-init -)' was not enough to get pyenv ready. In the official pyenv repository more configurations for specific shell and OS were added: https://github.com/pyenv/pyenv/blob/master/README.md#basic-github-checkout. 

It solves the issue in case anyone has: "pyenv not found" after testing pyenv installation.

edithturn committed:
add problem and solution documentation for pyenv not found

edithturn committed:
add problem and solution documentation for pyenv not found

o-nikolas reviewed (commented):
Someone beat you to it :)  https://github.com/apache/airflow/pull/19100/files

Although your url links more directly to the relevant location. So you could rebase from main and at least update that.

meysam81 committed:
docs: reorder imports in tutorials 🎨 (#19035)

Co-authored-by: Kaxil Naik <kaxilnaik@gmail.com>

josh-fell committed:
Updating Apache HDFS title in index doc (#19169)

enima2684 committed:
Add support of placement in the DockerSwarmOperator (#18990)

avkirilishin committed:
Add explicit session parameter in PoolSlotsAvailableDep (#18875)

robinedwards committed:
Fix task instance modal in gantt view (#19258)

nikie committed:
Update BaseOperator type hints for retry_delay, max_retry_delay, dag. (#19142)

uranusjr committed:
Remove redundant Union of one item (#19297)

Bowrna committed:
Typo fix in TESTING.rst (#19216)

ignaski committed:
Fixing ses email backend (#18042)

uranusjr committed:
Skip triggerer in 'breeze start-airflow' if on 3.6 (#19305)

The triggerer does not work on 3.6, so there's no point showing a dead
pane.

dstandish committed:
Clarify rat test guidance in release check docs (#19296)

* Clarify rat test guidance in release check docs

In the context of the rat tests there are two binaries that need unpacking -- the rat test jar and the airflow release.  This change clarifies the references to the airflow binaries.

josh-fell committed:
Expanding ``.output`` operator property information in TaskFlow tutorial doc (#19214)

edithturn committed:
adding direct link to configure pyenv in specific shell

edithturn committed:
adding specific section of the README to configure shell

edithturn committed:
add README.md

edithturn committed:
CONTRIBUTORS_QUICK_START.rst

edithturn committed:
Add specifi url to configure pyenv on shell on CONTRIBUTORS_QUICK_START.rst

edithturn committed:
Add specifi url to configure pyenv on shell on CONTRIBUTORS_QUICK_START.rst

edithturn committed:
Merge branch 'add-documentation-pyenv-nofound' of https://github.com/edithturn/airflow into add-documentation-pyenv-nofound

edithturn committed:
Add specifi url to configure pyenv on shell
